addons/cookoff/functions/fnc_cookOffEffect.sqf

```diff
@@ -37,7 +37,7 @@ if (isServer) then {
     if (_ring) then {
         private _intensity = 6;
         private _radius = 1.5 * ((boundingBoxReal _obj) select 2);
-        [QEGVAR(fire,addFireSource), [_obj, _radius, _intensity, _obj]] call CBA_fnc_localEvent;
+        [QEGVAR(fire,addFireSource), [_obj, _radius, _intensity, str _obj + "_cookoff"]] call CBA_fnc_serverEvent;
```

Comment:
Not sure, would this be any better?
I think technically a vic could be named differently of different machines.
```suggestion
        [QEGVAR(fire,addFireSource), [_obj, _radius, _intensity, hashValue _obj + "_cookoff"]] call CBA_fnc_serverEvent;
```